internal/pkg/cli/project_init.go

```diff
@@ -21,14 +23,21 @@ import (
 type InitProjectOpts struct {
 	ProjectName string
 
+	identity     identityService
 	projectStore archer.ProjectStore
 	ws           archer.Workspace
 	prompt       prompter
 }
 
 // NewInitProjectOpts returns a new InitProjectOpts.
 func NewInitProjectOpts() (*InitProjectOpts, error) {
+	defaultSession, err := session.Default()
+	if err != nil {
+		return nil, err
```

Comment:
nit: Do we need to wrap this error or is the message descriptive?

Reply:
Will add in follow up :D